package.json

```diff
@@ -4,8 +4,6 @@
   "description": "Interactive shell for managing Docker containers",
   "main": "index.js",
   "scripts": {
-    "test": "yarn run snyk:test && yarn run lint",
```

Comment:
let's make sure we leave the test run script to run lint

Reply:
👍  Updated the PR.